src/include/daos_task.h

```diff
@@ -644,10 +644,10 @@ typedef struct {
  *   to allocate multiple instances of this data structure.
  */
 typedef struct {
-	/** Object open handle */
-	daos_handle_t		oh;
 	/** Transaction open handle. */
 	daos_handle_t		th;
+	/** Object open handle */
+	daos_handle_t		oh;
```

Comment:
style: for sake of consistency, maybe make the same change for listing and key_query task args even though it's not required.

Reply:
I make this change is mainly because the tx_commit task buffer will be reused for single update/punch request when commit the distributed transaction. I also considered to change another non-affected cases, but there are seems too many candidates, such as list/query and kinds of array cases. So only changed update/punch cases finally.

> looks good to me with Liang's suggestions.

As for Liang's suggestion, consider current one cycle CI test may take 2 days, I prefer to fix them in the subsequent patch PR#2340.